src/gui/mainwindow.h

```diff
@@ -103,7 +103,7 @@ class MainWindow: public QMainWindow
     void showNotificationBaloon(QString title, QString msg) const;
 
 private slots:
-    void toggleVisibility(QSystemTrayIcon::ActivationReason e = QSystemTrayIcon::Trigger);
+    void toggleVisibility(QSystemTrayIcon::ActivationReason reason = QSystemTrayIcon::Trigger);
```

Comment:
>You added it in .cpp file, but don't do it here, so you have different signatures now.

But it's the same to the compiler.
and I remember my code got rejected by adding `const` to a variable in function declaration before...
to clear this up, are there any rules you have in mind?